### DISCUSSION TITLE
Make shows "Progress" tab less redundat and confusing

### DISCUSSION THREAD

**Discussion by seldon1000** [trakt/showly#358] [Ideas / Feature Requests]
"Progress" tab needs to be straight forward, understandable and easy to use. As of now, the tab seems to be pretty redundant, with "Upcoming" section and "Calendar doing pretty much the same thing. See the screenshots below (Calendar - Upcoming):

### **Now**

![Schermata del 2021-01-06 22-11-34](https://user-images.githubusercontent.com/55358113/103820198-2f964780-506c-11eb-863d-77b7e5512817.png)

It would me nicer to have just "Calendar" doing this job and remove "Upcoming" section completely, which makes it confusing to navigate through "Progress" tab. Also, it would be even better to make a clear separation between shows you are watching and shows you haven't started yet. I made a pretty simple and ugly mock-up, but it should give an idea. Here (Calendar - Progress):

### **After** (more or less)

![Schermata del 2021-01-06 22-26-55](https://user-images.githubusercontent.com/55358113/103821366-4fc70600-506e-11eb-81b0-3ef64555465f.png)

**Comment by michaldrabik** [comment]
Hey. I've actually been thinking about this before and the one issue I have with this is that whenever you have a new episode upcoming the show and all that progress info would disappear from Progress tab.

I'd find this confusing that show suddenly disappear after I mark an episode when the progress is still not 100%.
Now you have a visual representation of the element moving down into Upcoming section so you know what happened.

Also Calendar tab is slighlty different because it shows premiere episodes. This means that you can for example be in the process of watching "American Gods" season 1 or 2 and you can still see that Season 3 premieres on Monday 11th.

**Reply by seldon1000** [reply]
But what's the point of keeping a show in Progress tab if you cannot actually progress because new episodes are not available? Once a new episode is out and you can actually progress, the show should be displayed into the tab, otherwise it seems just superfluous. When I use the tab I feel overwhelmed by the amount of items in there, while most of them are simply displaying some infos and are not "usable".

I believe those infos can be shown in Calendar too, with a bit of restyling.

**Reply by michaldrabik** [reply]
I think the best option here is to make hiding this section optional via Settings. To me personally and many users having this section makes sense but I also see the point of reasoning against it in same cases.

**Reply by seldon1000** [reply]
Yeah, it makes sense in the absence of a better solution. Maybe make it a dropdown list, which can be compressed and expanded when needed in-place without reaching the settings?

Also, separating shows you haven't started from the rest is big step forward for clarity. Maybe make it a dropdown list too? So that these sections can be hidden and shown independently.
